image/base/buildllvm.sh

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/env sh
+
+# WARNING!
+# This script makes use of the `local' keyword. Although it is not POSIX
+# compliant, all shells on Linux (dash, ash, bash, zsh, ...) support it.
+
+runpath="$(realpath "$(dirname "$0")")"
+
+version=3.2
+baseurl="http://llvm.org/releases/$version"
+basedir=/opt/s2e/llvm
+prefix="$basedir/llvm-${version}-native"
+
+cores=1
+test -e /proc/cpuinfo && cores=$(grep -c '^processor' /proc/cpuinfo)
+
+# TODO: why is the patched LLVM called 'native'?
```

Comment:
Because there are two LLVM versions built:
* The native one includes clang + LLVM and is used to build S2E itself on the host.
* The target (x86) one only includes LLVM and is used for LLVM code generation during symbolic execution.